dom.bs

```diff
@@ -2583,8 +2583,8 @@ steps:
 <h4 id=mutation-algorithms>Mutation algorithms</h4>
 
 <p>To
-<dfn export for=Node id=concept-node-ensure-pre-insertion-validity>ensure pre-insertion validity</dfn>
-of a <var>node</var> into a <var>parent</var> before a <var>child</var>, run these steps:
+<dfn export for=Node id=concept-node-ensure-pre-insert-validity>ensure pre-insert validity</dfn> of
```

Comment:
Let's keep the ID unchanged (no need to add an alias either, I like that the IDs sometimes leak a bit of editing history).